src/types.ts

```diff
@@ -32,16 +32,19 @@ export interface MarketplaceCreatorPayload {
 
 export interface MarketplaceAddressPayload {
   owner?: string
-  auctionHouse: string
   store?: string
   storeConfig?: string
 }
+interface MarketplaceAuctionHousePayload {
+  address: string
+}
 export interface MarktplaceSettingsPayload {
   meta: MarktetplaceMetaPayload
   theme: MarketplaceThemePayload
   creators: MarketplaceCreatorPayload[]
   subdomain: string
   address: MarketplaceAddressPayload
+  auctionHouses?: MarketplaceAuctionHousePayload[]
```

Comment:
This should either be an empty list or filled with auctionHouses we shouldn't make it nullable.